dist/releases/4.16/config.toml

```diff
@@ -1,3 +1,7 @@
+certified_distributions = [
+  "Red Hat Enterprise Linux release 9.2 (Plow)",
+]
+
```

Comment:
Version-specific configuration is an add-on to system-wide one (IOW version-specific one is merged into the global one). Here you add the same string to both global and version-specific configs, which is redundant.

Having said that, what might be missing is the merging code (haven't checked).

Reply:
In other words, if something is in the global config, it does not have to be repeated.

Reply:
For this to work, you need to amend `func (c *ConfigFile) Add(add *ConfigFile) error` using `appendUniq` (and maybe add appropriate test cases).

Reply:
Nice -thanks for the recommendations. I worked that into the latest revision.